src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/ExtractOriginalAlignmentRecordsByNameSpark.java

```diff
@@ -55,31 +51,25 @@ public boolean requiresReads() {
     @Override
     protected void runTool( final JavaSparkContext ctx ) {
 
-        final Broadcast<HashSet<String>> namesToLookForBroadcast = ctx.broadcast(parseReadNames());
+        final Broadcast<Set<String>> namesToLookForBroadcast = ctx.broadcast(parseReadNames());
 
         final JavaRDD<GATKRead> reads =
-                getUnfilteredReads().repartition(80)
-                        .filter(read -> namesToLookForBroadcast.getValue().contains(read.getName())).cache();
-        localLogger.info("Found these many alignments: " + reads.count());
-
+                getUnfilteredReads().filter(read -> namesToLookForBroadcast.getValue().contains(read.getName())).cache();
```

Comment:
I wonder what is the cpu/walltime impact of this cache just for the sake of outputting the count in the INFO message below. Is it really worth it?

Reply:
There's a `count` after the write, so I prefer to cache it.

Reply:
Yeah... but is it worth the extra CPU/Wall time... the inside you get by outputting the count 
 is it worth the extra CPU/Wall time? In any case that one was jus a comment, no change needed.